collections/app/model/Node.scala

```diff
@@ -22,8 +22,7 @@ object Node {
           // use the T at this level or an empty node to hold children
           Node(currentSlug, loop(children, thisFullPath), thisFullPath, thisLevel.headOption)
         }
-        .sortBy(_.fullPath.head)
-        .sortBy(_.children.isEmpty)
+        .sortBy(node => (node.children.isEmpty, node.basename))
```

Comment:
It does as we're [sorting within the loop](https://github.com/guardian/grid/blob/e15305800f96db81f9295f55dc00b5e8811b8859/collections/app/model/Node.scala#L11-L26).